gradlew.bat

```diff
@@ -1,17 +1,24 @@
 @rem
-@rem Copyright 2015 the original author or authors.
+@rem Copyright © 2021 Lucy Poulton
```

Comment:
I don't own copyright over this

Reply:
Oh shoot! Massive oversight! Reverted in https://github.com/lucyy-mc/Squirtgun/pull/29/commits/d0475edd98d18594eb7e3775f4ca0177f4d1852a :sweat_smile: